lib/phoenix/live_dashboard/helpers/live_helpers.ex

```diff
@@ -277,4 +281,17 @@ defmodule Phoenix.LiveDashboard.LiveHelpers do
       socket
     end
   end
+
+  @doc """
+  Current path and maybe it updates the params
+  """
+  def self_path(socket, menu, new_params_or_fun \\ nil) do
```

Comment:
Instead of calling it `self_path`, maybe we should make it be `live_dashboard_path`? We would probably have to remove the default argument from `live_dashboard_path/4` though - which is probably fine.

Reply:
Done. Because the menu is not "yet" a struct, I prefer to be more strict in the pattern matching to avoid confusions if someone try to use the old `live_dashboard_path/3` version